src/filters/dates/relative/relativeDates.ts

```diff
@@ -0,0 +1,73 @@
+import { ParsedMatchSchema } from 'serina.schema';
+import { DateTime } from 'luxon';
+import { DATES } from '../dates.constants';
+import RELATIVE_DATES, { RELATIVE_ADVERB } from './relativeDates.constants';
+import { parseMatches, contains, remove, matchPattern } from 'utils';
+
+export default class RelativeDates {
+    static parseText(text: string): ParsedMatchSchema[] {
+        const pattern = `(${DATES.FILLER_WORDS})?${RELATIVE_DATES.ANY}`;
+        const matches = matchPattern(text, pattern, false);
+
+        if (!matches) return null;
+
+        return matches.map(match => {
+            const dateTimeObj = this.convertMatchToDateObj(match);
+            return parseMatches(text, match, dateTimeObj);
+        });
+    }
+
+    static timeUnitToString(matchAgainst: string): string {
+        for (const key in RELATIVE_DATES.TIME_UNITS) {
```

Comment:
Remember how we said we are not going to use `for in` loops lol

Reply:
What is the problem with it? We need to loop over object properties and that's what it does.

Reply:
Not ES5, that's all

Reply:
I'll go through it all and change it later